package.json

```diff
@@ -69,7 +69,7 @@
     "ts-helpers": "^1.1.1",
     "tslint": "^5.9.1",
     "tslint-microsoft-contrib": "^5.0.3",
-    "typescript": "^2.7.2",
+    "typescript": "~2.7.2",
```

Comment:
Any higher version supported? Not a big deal either way.

Reply:
nope, the compiler CLI requires 2.7 and errors if a higher version is present.